tensorboard/uploader/proto/server_info.proto

```diff
@@ -92,3 +102,9 @@ message PluginControl {
   // stored in the the `SummaryMetadata.plugin_data.plugin_name` proto field.
   repeated string allowed_plugins = 1;
 }
+
+message UploadLimits {
+  // The maximum allowed size for blob uploads.
+  // If this is 0, no blobs should be uploaded.
+  int64 max_blob_size = 1;
+}
```

Comment:
Hmm, I approved thinking that lint would catch this, but it didn’t,
somehow: missing newline at end of file.

(I’ll look into why.)